integration/kubernetes/README.md

```diff
@@ -67,11 +66,18 @@ bash helm-generate.sh multi-embedded
 
 ```
 
-For *singleMaster-hdfsJournal* the script relies on the `config.yaml` file you just created under the hood.
+For all modes the script relies on the `config.yaml` file in the corresponding template directory.
```

Comment:
@madanadit I changed "target directory" to "the corresponding template directory". Do you think it clarified better?